build-test.xml

```diff
@@ -8863,6 +8863,64 @@ go</echo>
 		</if>
 	</target>
 
+	<target name="check-mbean-attributes">
+		<beanshell>
+			<![CDATA[
+				import com.liferay.portal.upgrade.internal.recorder.UpgradeRecorder;
+
+				import java.lang.Object;
+				import java.lang.management.ManagementFactory;
+
+				import java.util.HashSet;
+				import java.util.Set;
+
+				import javax.management.*;
+				import javax.management.MBeanServer;
+				import javax.management.ObjectName;
+
+				MBeanServer mbs = ManagementFactory.getPlatformMBeanServer();
+
+				String mbsToString = mbs.toString();
+
+				for (int i = 0; i < 30; i++) {
+					Thread.sleep(100);
+
+					mbs = ManagementFactory.getPlatformMBeanServer();
+
+					if(!(mbsToString.equals(mbs.toString()))) {
+						i = 0;
+					}
+				}
+
+				System.out.println("Result Server: " + mbs);
+
+				Set mbeanNames = mbs.queryNames(null, null);
+
+				for (Iterator i = mbeanNames.iterator(); i.hasNext();) {
+					ObjectName mbeanName = (ObjectName)i.next();
+					System.out.println("MBean: " + mbeanName);
+				}
+
+				ObjectName objectName = new ObjectName("com.liferay.portal.upgrade:classification=upgrade," + "name=UpgradeManager");
+
+				// Verifica se o MBean está registrado
+				if (mbs.isRegistered(objectName)) {
+
+					// Obtém o valor do atributo "result"
+					String result = (String)mbs.getAttribute(objectName, "Result");
+					System.out.println("Result: " + result);
+
+					// Obtém o valor do atributo "type"
+					String type = (String)mbs.getAttribute(objectName, "Type");
+					System.out.println("Type: " + type);
+				}
+				else {
+					System.out.println("UpgradeManager MBean não está registrado");
+				}
+			]]>
```

Comment:
If the Mbean was registered, should conclude the 3 step, get the type and result